app/ui-flextab/client/tabs/membersList.js

```diff
@@ -35,9 +36,14 @@ Template.membersList.helpers({
 		const room = ChatRoom.findOne(this.rid);
 		const roomMuted = (room != null ? room.muted : undefined) || [];
 		const userUtcOffset = Meteor.user() && Meteor.user().utcOffset;
+		const hierarchy = ['admin', 'owner', 'leader', 'moderator', 'Rocket.Chat team'];
```

Comment:
I think the roles should not be hardcoded. Also, there is no clear definition of hierarchy between roles besides how many/which permissions are granted for a role.

Reply:
@tassoevan I was also afraid that hardcoding might be a bad idea. Could you please suggest what should I do instead?

Reply:
Thoughts, @RocketChat/core?